rpcs3/Emu/system_config_types.cpp

```diff
@@ -40,10 +40,13 @@ void fmt_class_string<video_resolution>::format(std::string& out, u64 arg)
 	{
 		switch (value)
 		{
-		case video_resolution::_1080: return "1920x1080";
-		case video_resolution::_720: return "1280x720";
-		case video_resolution::_480: return "720x480";
-		case video_resolution::_576: return "720x576";
+		case video_resolution::_1080p: return "1920x1080";
+		case video_resolution::_1080i: return "1920x1080i";
+		case video_resolution::_720p: return "1280x720";
+		case video_resolution::_480p: return "720x480";
+		case video_resolution::_480i: return "720x480i";
+		case video_resolution::_576p: return "720x576";
```

Comment:
I don't think we support interlaced video.

Reply:
It's now removed from the dropdown, but still useable via config.yml